tests/conftest.py

```diff
@@ -42,10 +42,10 @@ def sensors():
     return tu.SENSORTAG_LIST
 
 
-@pytest.fixture
+@pytest.fixture(scope="session")
```

Comment:
This seems unsafe, one temp directory for the entire test session?

Reply:
For some tests this is useful. The pytest builtin `tmpdir` fixture is function scoped, so we could use that in instances where we want things to get flushed. From what I've seen, this doesn't seem to be problem with existing tests, but I can make doubly sure :+1:

Reply:
ya, but that's nothing to preclude it from being a weird problem in the future. Is it possible to just add the tempdir on creation at the specific fixture which requires it for the session?

Reply:
Definitely. This just seemed a tad bit cleaner, but I'm down with that if that seems best if it may be problematic.

Reply:
Cool, and I think us implementing our own `tmp_dir` was merely an oversight from the pytest builtin :+1: